components/color-palette/index.js

```diff
@@ -65,8 +65,17 @@ export default function ColorPalette( { colors, disableCustomColors = false, val
 					renderContent={ () => (
 						<ChromePicker
 							color={ value }
-							onChangeComplete={ ( color ) => onChange( color.hex ) }
-							disableAlpha
+							onChangeComplete={ ( color ) => {
+								let colorString;
+								if ( typeof color.rgb === 'undefined' || color.rgb.a === 1 ) {
+									colorString = color.hex;
+								} else {
+									const { r, g, b, a } = color.rgb;
+									colorString = 'rgba(' + [ r, g, b, a ].join( ',' ) + ')';
+								}
+								onChange( colorString );
```

Comment:
Seems there's a code path here where `colorString` is `undefined`, should we test for its truthiness before running `onChange`?

Reply:
> Seems there's a code path here where `colorString` is `undefined`

What's the code path?

Reply:
Oh wait, there isn't, but because it's defined as `undefined` above it sort of read like maybe there could be.

I actually see why it's this way, never mind me, all good!